models.py

```diff
@@ -307,9 +307,12 @@ def get_task_state(self):
 
     def get_default_impact_title(self):
         layer_name = '%s on %s' % (
-            self.hazard_layer.name,
-            self.exposure_layer.name
+            self.hazard_layer.title,
+            self.exposure_layer.title
         )
+        if self.aggregation_layer:
+            additional_name = 'around %s' % self.aggregation_layer.title
+            return layer_name + ' ' + additional_name
```

Comment:
In fact, considering this end result, it is better to do it like this (for this whole method):

```suggestion
            title_dict = {
                'hazard': self.hazard_layer.title,
                'exposure': self.exposure_layer.title
            }
            if self.aggregation_layer:
                title_template = _('{hazard} on {exposure} around {aggregation}')
                title_dict['aggregation'] = self.aggregation_layer.title
            else:
                title_template = _('{hazard} on {exposure}')
             return title_template.format(**title_dict)
                
```

Reply:
This way, the translator has context and shuffle the variable around if needed.